src/kinematics_plugin.cpp

```diff
@@ -236,37 +192,34 @@ struct BioIKKinematicsPlugin : kinematics::KinematicsBase {
 
     link_names = tip_frames_;
 
-    // for(auto& n : joint_names) LOG("joint", n);
-    // for(auto& n : link_names) LOG("link", n);
-
     // bool enable_profiler;
-    lookupParam("profiler", enable_profiler, false);
+    getRosParam("profiler", enable_profiler, false);
     // if(enable_profiler) Profiler::start();
 
-    robot_info = RobotInfo(robot_model);
+    robot_info = RobotInfo(robot_model_);
 
-    ikparams.robot_model = robot_model;
+    ikparams.robot_model = robot_model_;
     ikparams.joint_model_group = joint_model_group;
 
     // initialize parameters for IKParallel
-    lookupParam("mode", ikparams.solver_class_name,
+    getRosParam("mode", ikparams.solver_class_name,
                 std::string("bio2_memetic"));
-    lookupParam("counter", ikparams.enable_counter, false);
-    lookupParam("threads", ikparams.thread_count, 0);
-    lookupParam("random_seed", ikparams.random_seed, static_cast<int>(std::random_device()()));
+    getRosParam("counter", ikparams.enable_counter, false);
+    getRosParam("threads", ikparams.thread_count, 0);
+    getRosParam("random_seed", ikparams.random_seed, static_cast<int>(std::random_device()()));
 
     // initialize parameters for Problem
-    lookupParam("dpos", ikparams.dpos, DBL_MAX);
-    lookupParam("drot", ikparams.drot, DBL_MAX);
-    lookupParam("dtwist", ikparams.dtwist, 1e-5);
+    getRosParam("dpos", ikparams.dpos, DBL_MAX);
+    getRosParam("drot", ikparams.drot, DBL_MAX);
+    getRosParam("dtwist", ikparams.dtwist, 1e-5);
 
     // initialize parameters for ik_evolution_1
-    lookupParam("no_wipeout", ikparams.opt_no_wipeout, false);
-    lookupParam("population_size", ikparams.population_size, 8);
-    lookupParam("elite_count", ikparams.elite_count, 4);
-    lookupParam("linear_fitness", ikparams.linear_fitness, false);
+    getRosParam("no_wipeout", ikparams.opt_no_wipeout, false);
+    getRosParam("population_size", ikparams.population_size, 8);
+    getRosParam("elite_count", ikparams.elite_count, 4);
+    getRosParam("linear_fitness", ikparams.linear_fitness, false);
 
-    temp_state.reset(new moveit::core::RobotState(robot_model));
+    temp_state.reset(new moveit::core::RobotState(robot_model_));
```

Comment:
```suggestion
    temp_state = std::make_unique<moveit::core::RobotState>(robot_model_);
```

Reply:
Added in [c7edb0d](https://github.com/PickNikRobotics/bio_ik/commit/c7edb0dc0e4c107831cbbc116cc3f9547bb98564)